packages/db/src/index.ts

```diff
@@ -1,10 +1,15 @@
-import { type Prisma, PrismaClient } from "@prisma/client"
+import { createRequire } from "node:module"
+import type { Prisma, PrismaClient } from "@prisma/client"
 import type { DefaultArgs } from "@prisma/client/runtime/library"
 
-export * as schemas from "./schemas"
+const require = createRequire(import.meta.url)
+const { Prisma: _Prisma, PrismaClient: _PrismaClient } = require("@prisma/client")
+export type * from "@prisma/client"
 
+export const PrismaRuntime = _Prisma
+export const PrismaClientRuntime = _PrismaClient
 export type DBClient = PrismaClient<Prisma.PrismaClientOptions, never, DefaultArgs>
 export const createPrisma = (databaseUrl: string): DBClient =>
-  new PrismaClient({
+  new _PrismaClient({
```

Comment:
Prisma does not generate ESM module code, so we have to createRequire() and require it here. For this reason, the `Prisma` export cannot be both the client and namespace (like the generated client does).

Therefore we have to distinguish between Prisma namespace (re-exported from the client), and the runtime values.

Reply:
This means the Dockerfile build was broken in #232, but we never noticed because CI never built them. We'd have to do this at some point anyways